deploy/operator.yaml

```diff
@@ -22,9 +22,7 @@ spec:
           imagePullPolicy: Always
           env:
             - name: WATCH_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
```

Comment:
removing these would me to also regenerate the CSV, right ?

Reply:
No, I think it'll be fine. Because the only installMode we support now is AllNamespaces, the current value from the fieldPath should come in as "", which is what we need. We need to be "", whether hardcoded (in operator.yaml, so that `kubectl apply -f deploy/` works) or taken from the OLM annotation (when the CSV is used). I hope that makes sense.